testing/file_test/autoupdate.cpp

```diff
@@ -43,6 +43,7 @@ FileTestAutoupdater::FileAndLineNumber::FileAndLineNumber(
       line_number(ParseLineNumber(line_number)) {}
 
 auto FileTestAutoupdater::CheckLine::RemapLineNumbers(
+    const llvm::DenseMap<llvm::StringRef, int>& file_to_number_map,
```

Comment:
Historically we used DenseMap, but should this code actually be shifting towards the newer Map structure?

Though I see you're just extracting code to a helper, so maybe out-of-scope for this PR.

Reply:
Yeah, I think this is out of scope.